lib/rules/key-spacing.js

```diff
@@ -98,6 +98,35 @@ module.exports = function(context) {
         afterColon = +!(options.afterColon === false); // Defaults to true
 
     /**
+     * Gets the first node on the same line with the provided node.
+     * @param {ASTNode} node The node to check find from
+     * @returns {ASTNode} the first node on the given node's line
+     */
+    function getFirstNodeInLine(node) {
+        var startLine, endLine, firstToken = node;
+
+        do {
+            node = firstToken;
+            firstToken = context.getTokenBefore(node);
+            startLine = node.loc.start.line;
+            endLine = firstToken ? firstToken.loc.end.line : -1;
+        } while (startLine === endLine);
+
+        return node;
+    }
+
+    function getFirstNodeBeforeColon(node) {
+        var prevNode;
+
+        while (node.type !== "Punctuator" || node.value !== ":") {
```

Comment:
They are double quotes and the linter did catch that :) I used single quotes first and just run the tests for this rule so missed it. Then CI yelled at me and then I fixed.